packages/ui/core-components/src/lib/unsorted/viz/map/PointMap.svelte

```diff
@@ -67,6 +67,9 @@
 	/** @type {boolean} */
 	export let legend = true;
 
+	/** @type {boolean} */
+	export let ignoreZoom = false;
```

Comment:
Should we use `toBoolean` here too?

Reply:
toBoolean is included in `Points` to handle it there. Does that work?

Reply:
Yes, as long as the `ignoreZoom` prop isn't used anywhere inside `PointMap`